x/auth/client/cli/multisign.go

```diff
@@ -0,0 +1,161 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/tendermint/crypto/multisig"
+	"github.com/tendermint/tendermint/libs/cli"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/keys"
+	crkeys "github.com/cosmos/cosmos-sdk/crypto/keys"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	authtxb "github.com/cosmos/cosmos-sdk/x/auth/client/txbuilder"
+)
+
+// GetSignCommand returns the sign command
+func GetMultiSignCommand(codec *amino.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "multisign <file> <name> <<signature>...>",
+		Short: "Generate multisig signatures for transactions generated offline",
+		Long: `Sign transactions created with the --generate-only flag that require multisig signatures.
+
+Read signature(s) from <signature> file(s), generate a multisig signature compliant to the
```

Comment:
I'm thinking maybe we can add a small example here.

Reply:
Done